testing/src/test/java/io/stargate/it/http/RestApiTest.java

```diff
@@ -649,6 +687,35 @@ public void health() throws IOException {
         .isEqualTo("UP");
   }
 
+  private void createComplexTable(String tableName) throws IOException {
+    TableAdd tableAdd = new TableAdd();
+    tableAdd.setName(tableName);
+
+    List<ColumnDefinition> columnDefinitions = new ArrayList<>();
+
+    columnDefinitions.add(new ColumnDefinition("pk0", "uuid"));
+    columnDefinitions.add(new ColumnDefinition("col1", "frozen<map<date, varchar>>"));
```

Comment:
are UDTs supported?
If so then we should also include:
- udt using collections
- collections of udts
- tuple with udts
- udts with tuples
- tuple with collection of udts

Reply:
Not yet